Tabloid/client/src/components/CategoryList.js

```diff
@@ -25,8 +25,8 @@ const CategoryList = () => {
       <div className="container">
         <div className=" mt-4 row justify-content-center">
           {categories.map((category) => (
-            <ListGroupItem><Category category={category} key={category.id} />
-            <Link to={`/categories/edit/${category.id}`}>Edit</Link>{" "}
+            <ListGroupItem key={`editlink--${category.id}`}><Category category={category} key={category.id} />
```

Comment:
Do you get a key error when the key attribute is removed from the Category component?